frontend/javascripts/oxalis/model/accessors/annotation_accessor.ts

```diff
@@ -97,23 +98,30 @@ export function getCombinedStats(tracing: Tracing): CombinedTracingStats {
 export function getCombinedStatsFromServerAnnotation(
   annotation: APIAnnotationInfo,
 ): CombinedTracingStats {
-  const aggregatedStats: TracingStatsHelper = {};
+  return aggregateStatsForAllLayers(
+    annotation.annotationLayers.map((annotation) => annotation.stats),
+  );
+}
 
-  for (const annotationLayer of annotation.annotationLayers) {
-    const { stats } = annotationLayer;
+export function aggregateStatsForAllLayers(
+  stats: Array<TracingStats | EmptyObject>,
+): CombinedTracingStats {
+  const aggregatedStats: TracingStatsHelper = {};
 
-    if ("treeCount" in stats) {
-      const { treeCount, nodeCount, edgeCount, branchPointCount } = stats;
+  for (const annotationLayerStats of stats) {
+    if (Object.keys(annotationLayerStats).includes("treeCount")) {
```

Comment:
why this change? :thinking: is `in` not okay?

Reply:
oh, this is a leftover from when somewhere in the process it wasnt working :)